### PR TITLE
fix(ZNTA-3): Hide download zip option when version is empty

### PR DESCRIPTION
**Pull request by djansen-redhat**
JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-3

Hides the "Download all as zip" option when there is nothing to download

## Reviewer tasks

The following is based on the
[Zanata Development Guidelines](https://github.com/zanata/zanata-server/wiki/Development-Guidelines).
As a reviewer, you should check the guidelines regularly for updates, and just
to refresh your memory.

The reviewer can use this list for reference to make sure
they have considered all the important points.

Use the checkboxes or not, whatever works best for you.

 - Code quality
   - [ ] Code passes style check (checkstyle/eslint)
   - [ ] Code is clear and concise
   - [ ] UI code is internationalised
   - [ ] Code has adequate comments where appropriate
   - Code is in an appropriate place
     - [ ] Classes are in appropriate packages
     - [ ] Code fits with class's responsibilities (SRP)
   - [ ] Configuration files are documented enough
   - If code is removed
     - [ ] No remaining code references the removed code
       - [ ] No orphaned rewrite rules
       - [ ] No orphaned config settings
     - [ ] No remaining comments refer to the removed code
     - [ ] Unused imports and libraries are removed
 - Testing
   - [ ] New code has tests
   - [ ] Changed code has tests
   - [ ] All tests pass
   - [ ] Test coverage stable or increased
 - Documentation
   - [ ] New features are documented
   - [ ] Docs removed for deleted features
   - [ ] Docs updated for all changed features
   - [ ] Developer/sysadmin docs updated if appropriate
 - If there are new dependencies
   - [ ] Does each new dependency pass all the
         [Considerations for new dependencies](https://github.com/zanata/zanata-server/wiki/Development-Guidelines#new-technologies-and-dependencies)?

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/103)

### CODE REVIEW

File: server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml [hunk 1]
```diff
@@ -201,7 +201,7 @@
           <ul class="dropdown__content js-dropdown__content" role="content"
             aria-labelledby="dropdownContent">
             <ui:fragment
-              rendered="#{versionHomeAction.zipFileDownloadAllowed}">
+              rendered="#{versionHomeAction.zipFileDownloadAllowed and versionHomeAction.getDocuments().size() > 0}">
```
Comment (aeng) [inline server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml]: Side note: you can use ...` and !versionHomeAction.getDocuments().isEmpty()` too.

**Comment by aeng** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-platform/103#-:-KYWvoku_ScoUStqD06B:bnfp4nl)*

File: server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml (hunk 1 above)
Comment (djansen-redhat) [inline server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml]: Which do you think is better?

Comment (aeng) [inline server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml]: im fine with both. Just a side note.

Comment (djansen-redhat) [inline server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml]: Okedoke, thanks

**Comment by spathare21** [comment]
verified

**Pull request merged**